tests/functional/example/test_scikit_learn.py

```diff
@@ -58,6 +52,7 @@ def test_orion_runs_script(monkeypatch):
 
 @pytest.mark.usefixtures("clean_db")
```

Comment:
You can remove `null_db_instances` and `clean_db` now, we don't use the mongodb backend in these tests anymore. :tada: :)

Reply:
I'm always confused with the current database setup.